docs/web/authentication.md

```diff
@@ -402,73 +442,73 @@ CodeChecker also supports OAuth-based authentication. The `authentication.method
 
         Currently there are only 2 options for this:
          * `username`
-         * `email`
+         * `email` (*default*)
 
         Expected output for each provider:
 
-        `github`
+        `github/v1`
```

Comment:
the provider name is "github" not github/v1

Reply:
Each available template, instead.